src/main/java/org/chrisle/netbeans/plugins/nbscratchfile/CreateScratchFile.java

```diff
@@ -1,158 +1,142 @@
-        }, false);
-
-        ((EventTarget) sourceElem).addEventListener("mouseout", (elem) -> {
-            sourceElem.setAttribute("style", oldCss);
-        }, false);
-    }
-
-    private void addHoverEffectToElements(NodeList sourceElements, String newCss, String oldCss) {
-        for (int i = 0; i < sourceElements.getLength(); i++) {
-            addHoverEffectToElement((Element) sourceElements.item(i), newCss, oldCss);
-        }
-    }
-
-    @Override
-    public void actionPerformed(ActionEvent e) {
-        showDialog();
-        initWebView();
-    }
-
-    private void initWebView() {
-        Platform.runLater(() -> {
-            webEngine.load(this.getClass().getResource("/org/chrisle/netbeans/plugins/nbscratchfile/ui/dist/index.html").toExternalForm());
-
-            try {
-                webEngine.getLoadWorker().stateProperty().addListener((ObservableValue<? extends State> ov, State oldState, State newState) -> {
-                    if (newState == State.SUCCEEDED) {
-                        JSObject win = (JSObject) webView.getEngine().executeScript("window");
-
-                        win.setMember("NbScratchFileViewModel", CreateScratchFile.this.viewModel);
-
-                        colorizeElement(webEngine.getDocument().getElementById("languageSearch"), String.format("background-color: %s; color: %s;", viewModel.getColor("TextField.background", false), viewModel.getColor("TextField.foreground", false)));
-                        colorizeElement((Element) webEngine.getDocument().getElementsByTagName("body").item(0), String.format("background-color: %s;", viewModel.getColor("Menu.background", false)));
-                        colorizeElement((Element) webEngine.getDocument().getElementsByTagName("ul").item(0), String.format("color: %s;", viewModel.getColor("Label.foreground", false)));
-
-                        addHoverEffectToElements(webEngine.getDocument().getElementsByTagName("li"), String.format("background-color: %s; color: %s;", viewModel.getColor("Menu.background", true), viewModel.getColor("Menu.foreground", true)), String.format("background-color: %s; color: %s;", viewModel.getColor("Menu.background", false), viewModel.getColor("Menu.foreground", false)));
-                    }
-                });
-            } catch (Exception ex) {
-                Exceptions.printStackTrace(ex);
-            }
-        });
-    }
-
-    public void showDialog() {
-        // try to use monitor, where the input focus is
-        // therefor get the topmost component based on the input focus
-        Component focusOwner = KeyboardFocusManager.getCurrentKeyboardFocusManager().getFocusOwner();
-
-        if (null != focusOwner) {
-            while (focusOwner.getParent() != null) {
-                focusOwner = focusOwner.getParent();
-            }
-        }
-
-        dialog.setLocationRelativeTo(focusOwner);
-        dialog.setVisible(true);
-    }
+package org.chrisle.netbeans.plugins.nbscratchfile;
```

Comment:
Sorry for the removal/addition. There is something wrong with end of lines - I couldn't get them right. At the end I used `dos2unix` command, but that is probably not correct either.